hw/top_earlgrey/data/ip/chip_clkmgr_testplan.hjson

```diff
@@ -0,0 +1,330 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: chip_clkmgr
+  testpoints: [
+    // CLKMGR tests:
+    {
+      name: chip_sw_clkmgr_idle_trans
+      desc: '''Verify the ability to turn off the transactional clock via SW.
+
+            Ensure that the clock to transactional units will be turned off after any activity
+            completes in the transactional IP.  Verify it is off via spinwait in hints_status CSR.
+            Verify that turning off this clock does not affect the other derived clocks.
+
+	    SiVal: No OTP or lifecycle dependencies. This tests already run in CW310.
```

Comment:
Does it requires cpu enable and / or clock calibration?

Reply:
You are right: it needs the CPU to be enabled, since it performs CSR writes. And in some others below.